core/src/main/java/io/grpc/transport/AbstractClientStream.java

```diff
@@ -125,8 +130,8 @@ protected void inboundHeadersReceived(Metadata.Headers headers) {
    *
    * @param frame the received data frame. Its ownership is transferred to this method.
    */
-  protected void inboundDataReceived(ReadableBuffer frame) {
-    Preconditions.checkNotNull(frame, "frame");
```

Comment:
Let me add:
$ git grep " checkNotNull" | wc -l
13
$ git grep "\.checkNotNull" | wc -l
110

The current style is overwhelmingly to use the full form. You are free to use the short form in code you write, but it does seem unhelpful to change all usages in the current code.